tests/testthat/test_utils.R

```diff
@@ -85,33 +85,33 @@ test_that("sym_limits has expected manual properties", {
     lower = log_lims[1],
     upper = log_lims[2]
   )
-  expect_equal(mean(log(log_lims)), log(pos_center))
+  expect_equal(mean(log10(log_lims)), log10(pos_center))
 })
 
 test_that("sym_limits agrees with manual expectation", {
   expect_equal(sym_limits(c(-5, 0.52, 8)), c(-8, 8))
   expect_equal(sym_limits(7, center = 2), c(-3, 7))
   expect_equal(sym_limits(c(-5, 0.52, 8), center = 5), c(-5, 15))
   expect_equal(
-    sym_limits(c(100, 2, 0.00002), center = 1, transform = "log"),
+    sym_limits(c(100, 2, 0.00002), transform = "log", center = 1),
     c(0.00002, 1 / 0.00002)
   )
   expect_equal(
-    sym_limits(c(100, 2, 0.00002), center = 2, transform = "log"),
+    sym_limits(c(100, 2, 0.00002), transform = "log", center = 2),
     c(0.00002, 2 * 2 / 0.00002)
   )
-  expect_equal(sym_limits(c(1 / 5), center = 1, transform = "log"), c(1 / 5, 5))
+  expect_equal(sym_limits(c(1 / 5), transform = "log", center = 1), c(1 / 5, 5))
 
   expect_equal(
-    sym_limits(c(0, 2, 0.00002), center = 2, transform = "log"),
+    sym_limits(c(0, 2, 0.00002), transform = "log", center = 2),
     c(0, Inf)
   )
   expect_equal(
-    sym_limits(c(0, 2, 0.00002), center = 1, transform = "log"),
+    sym_limits(c(0, 2, 0.00002), transform = "log", center = 1),
```

Comment:
```suggestion
    sym_limits(c(0, 2, 0.00002), transform = "log"),
```

Let's verify our new default works as intended.